Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -110,7 +110,7 @@ fetch() {
   then
     sha="$(sha256sum "$CACHED_LOCATION" | cut -d' ' -f1)"
   # Ruby 1.8.2's vendored Ruby has broken SHA256 calculation pre-G4e
```

Comment:
Thank you! Probably makes sense to update this comment, too:

```suggestion
  # Ruby 1.8.2's vendored Ruby has broken SHA256 calculation on several PowerPC CPUs
```